src/schedule.rs

```diff
@@ -295,7 +295,7 @@ impl Schedule {
 }
 
 impl Display for Schedule {
-    fn fmt(&self, f: &mut Formatter) -> FmtResult {
+    fn fmt(&self, f: &mut Formatter<'_>) -> FmtResult {
```

Comment:
I'm surprised that `rustfix` prefers to make these elisions explicit since that makes things more verbose. If you happen to have a link explaining the motivation, I'd be curious to learn more.

Reply:
I can't remember if this was explained in a blog post but if found these links 
https://doc.rust-lang.org/nightly/reference/lifetime-elision.html#lifetime-elision-in-functions
https://github.com/rust-lang/rfcs/blob/master/text/2115-argument-lifetimes.md#the-wildcard-lifetime